optimade/models/entries.py

```diff
@@ -91,8 +91,9 @@ class EntryResource(Resource):
 - **Type**: string.
 - **Requirements/Conventions**:
 
-  - **Response**: REQUIRED in the response unless explicitly excluded.
-  - **Query**: MUST be a queryable property with support for all mandatory filter operators.
+  - **Support**: MUST be supported by all implementations, MUST NOT be :val:`null`.
```

Comment:
```suggestion
  - **Support**: REQUIRED, MUST NOT be :val:`null`.
```
I'm wondering whether even the  `MUST NOT be null` is even necessary in this case (?)

Reply:
We should put it into the model actually, making it so that it raises if it's `None`.

Reply:
Do you think we should change the wording of the spec as you suggest, to make it from this specific server's point-of-view?